com.gitlab.maevemi.publictransport.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "com.gitlab.maevemi.publictransport",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "publictransport",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
build-args for network acces are not allowed. You have to either generate a manifest of the cargo dependencies 
(see https://github.com/flatpak/flatpak-builder-tools) or use vendored crates.